src/services/AudioMixer.cxx

```diff
@@ -218,9 +364,37 @@ void AudioMixer::loadSoundtrack(Iterator begin, Iterator end, CallbackType creat
           throw RuntimeError(ErrorMsg);
         }
         m_Soundtracks.emplace_back(createSoundtrack(name, chunk));
-        for(AudioTrigger trigger : soundtrack.triggers)
-          m_Triggers[trigger].emplace_back(name);
-    }
+        
+        #ifdef USE_OPENAL_SOFT
+      
+      //initialize buffer
+	  alGenBuffers(1, &m_Soundtracks.back()->buffer);
+	  assert(alGetError() == AL_NO_ERROR && "Could not create buffers");
```

Comment:
Replace by an exception
```suggestion
	  ALenum errorCode = alGetError();
	  if(errorCode != AL_NO_ERROR)
	    throw RuntimeError("Could not create buffers: Error " + std::to_string(errorCode));
```

Reply:
Resolved. Throwing an exception instead of asserting.